doc/flame_fire_atlas/fire_atlas.md

```diff
@@ -0,0 +1,96 @@
+# FireAtlas
```

Comment:
The `FireAtlas` and `flame_fire_atlas` are two different packages, and I think we should make a more clear distinction between them.

Generally, when writing this kind of Overview doc, it's good to imagine yourself in the shoes of a potential reader. The reader here probably knows nothing or very little about either `flame_fire_atlas` or `FireAtlas`, and the job of this document is to explain what those packages are, what do they do, and whether they would be useful to the reader.

For example, the "installation" section shouldn't be up front: it's like trying to convince someone to buy a product without them seeing it first.

Reply:
Agree.
On another note I think that we should also consolidate the docs for the bridge package with the docs for the package that the bridge leads to, because currently they are in different sections of the docs. That shouldn't be solved by this PR though.

Reply:
should i give title FlameFireAtlas ?

i saw on other bridging docs that they were not having flame prefix on heading so i just put it without prefix

Reply:
I believe it's perfectly fine to name the page as the package: `flame_fire_atlas`. We do that in the currently published docs: https://docs.flame-engine.org/1.4.0/flame_forge2d/flame_forge2d.html